sys/include/net/hdlc/fcs.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos <jialamos@uc.cl>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_hdlc_fcs    FCS calculation
```

Comment:
It's a different polynomial (see [this]). Maybe add another sys_checksum_crc function?

[this]:http://www.nongnu.org/avr-libc/user-manual/group__util__crc.html#ga37b2f691ebbd917e36e40b096f78d996